packages/panzer/adapters-stk/src/Panzer_STK_SetupLOWSFactory.cpp

```diff
@@ -105,31 +105,35 @@ namespace panzer_stk {
       reqHandler_local = Teuchos::rcp(new Teko::RequestHandler);
     #endif
 
+    auto stk_conn_manager = Teuchos::rcp_dynamic_cast<panzer_stk::STKConnManager>(conn_manager,true);
+
 #ifndef PANZER_ORDINAL64_IS_INT
```

Comment:
Do all of these if statements get removed after we update the DOFManager/GlobalIndexer?

Reply:
yes!!!